tests/nested/manual/fde-on-classic/mk-image.sh

```diff
@@ -61,23 +61,21 @@ install_data_partition() {
     set -x
     local DESTDIR=$1
     local CACHE=$2
-    # just some random date for the seed label
-    local SEED_LABEL=20220617
+    local SEED_LABEL
+    SEED_LABEL="$(date +%Y%m%d)"
```

Comment:
nitpick: I actually would prefer a fixed date to make sure the test is reproducible

Reply:
Sure thing - I updated the code now